code/modules/atmospherics/environmental/LINDA_system.dm

```diff
@@ -1,74 +1,76 @@
-/atom/var/CanAtmosPass = ATMOS_PASS_YES
-/atom/var/CanAtmosPassVertical = ATMOS_PASS_YES
+/atom/var/can_atmos_pass = ATMOS_PASS_YES
+/atom/var/can_atmos_pass_vertical = ATMOS_PASS_YES
```

Comment:
You gonna path these like you did the uh, air movement ones?

Reply:
done this and the turf ones